pandas/io/parsers.py

```diff
@@ -1770,8 +1772,9 @@ def _cast_types(self, values, cast_type, column):
                 values = astype_nansafe(values, cast_type,
                                         copy=True, skipna=True)
             except ValueError:
-                raise ValueError("Unable to convert column %s to "
-                                 "type %s" % (column, cast_type))
+                raise ValueError(
+                    "Unable to convert column {column} to type {type}".format(
```

Comment:
Does using `type` here present any potential conflict with the built-in? Maybe renamed to `type_` to err on the side of caution?